docs/rackhd/workflow_progress_notification.rst

```diff
@@ -119,16 +119,42 @@ Besides task started and finished events, a time-consuming task is not proper to
 
 **OS installation task progress**
 
-As a typical long-run task, OS installation task progress can not be easily measured. As a compromise, RackHD creates some milestones at important timeslot of installation process and divides the task into several sub-tasks.
-
-Take CentOS installation for example, 4 milestones are created for this task:
-
-- Profile downloaded: installation task started
-- Kernel downloaded: installer kernel downloaded
-- Installation started: installer started
-- Installation finished: installation and configuration completed
-
-With completion of each subtask, RackHD will send progress information to user with `value` varies from 1 to 4 and `maximum` fixed with 4;
+As a typical long-run task, OS installation task progress can't be easily measured. As a compromise, RackHD creates some milestones at important timeslot of installation process and divides OS install task into several sub-tasks.
+
+Below table includes descriptions for all existing RackHD OS installation milestones:
+
+=================== ==============================================================================================
+Milestone name      Milestone description
+=================== ==============================================================================================
+requestProfile      Enter ipxe and request OS installation profile. Common milestone for all OSes.
+enterProfile        Enter profile, start to download installer. Common milestone for all OSes.
+startInstaller      Start installer and prepare installation. Common milestone for all OSes.
+preConfig           Enter Pre OS configuration.
+startSetup          Net use Windows Server 2012 and start setup.exe. Only used for Windows Server.
+installToDisk       Execute OS installation. Only used for CoreOS.
+startPartition      Start partition. Only used for Ubuntu.
+postPartitioning    Finished partitioning and mounting, start package installation. Only used for SUSE.
```

Comment:
@yyscamper  It's better to use `postPartition` here to keep aligned with `startPartition`.